proto/regen/data/v1/types.proto

```diff
@@ -151,20 +151,25 @@ message ContentEntry {
   // hash is the content hash of the anchored data.
   ContentHash hash = 1;
 
-  // iri is the content IRI of the anchored data.
+  // iri is the IRI of the anchored data.
   string iri = 2;
 
-  // timestamp is the Timestamp of the anchored data.
+  // timestamp is the timestamp of the anchored data.
   google.protobuf.Timestamp timestamp = 3;
 }
 
-// AttestorEntry is a attestor entry wrapping a attestor address and timestamp
+// AttestorEntry is an attestor entry including the attestor, the IRI, and the
+// timestamp.
 message AttestorEntry {
-  // attestor is the address of the attestor.
-  string attestor = 1;
+  // iri is the IRI of the data attested to.
```

Comment:
```suggestion
  // IRI of the data attested to.
```

Reply:
This is not the convention for writing comments. We always start with the name of the field.